receiver/faroreceiver/receiver.go

```diff
@@ -0,0 +1,212 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package faroreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/faroreceiver"
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	faro "github.com/grafana/faro/pkg/go"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componentstatus"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+	"go.opentelemetry.io/collector/receiver/receiverhelper"
+	"go.uber.org/zap"
+
+	farotranslator "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/faro"
+)
+
+const faroPath = "/"
```

Comment:
if I understand the faro [spec](https://github.com/grafana/faro/blob/main/spec/gen/faro.gen.yaml) correctly, should this support receiving data on /collect/{appKey}?

Reply:
agree, let's change the spec. It should be possible to use any URL